digits/workspaces.py

```diff
@@ -0,0 +1,11 @@
+from urlparse import urlparse, parse_qs
+
+def get_workspace(url):
+	try:
+		workspace_name = parse_qs(urlparse(str(url)).query, keep_blank_values=True)["workspace"][0].encode('utf-8')
```

Comment:
1. Rather than parsing the URL, let's use `flask.request.args`: https://github.com/NVIDIA/DIGITS/blob/f9ad5ced48/digits/utils/routing.py#L40-L41
2. Let's move this into `utils.routing` rather than creating a new file
3. Please use 4 spaces for indentation instead of tabs (here and elsewhere): https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces